model/version.go

```diff
@@ -601,6 +605,98 @@ func GetVersionsWithOptions(projectName string, opts GetVersionsOptions) ([]Vers
 	return res, nil
 }
 
+// ConstructManifest will construct a manifest from the given project and version.
+func ConstructManifest(v *Version, proj *Project, projectRef *ProjectRef, settings *evergreen.Settings) (*manifest.Manifest, error) {
```

Comment:
It looks to me like this is only used in model packages now in which case it can be private